test/jdk/javax/swing/text/html/StyleSheet/bug8257665.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * <Copyright>
+ */
+
```

Comment:
You can copy the copyright from any of the test present in this directory, just change the year to 2021